pages/common/docker-service.md

```diff
@@ -0,0 +1,28 @@
+# docker service
+
+> Manage services on docker daemon.
+> More information: <https://docs.docker.com/engine/reference/commandline/service/>.
+
+- List services on docker daemon:
+
+`docker service ls`
+
+- Create a new service:
+
+`docker service create --name {{service_name}} {{docker_image_name}}:{{docker_image_version}}`
```

Comment:
This command is not for beginners with Docker. I think we can simplify the variable names:
```suggestion
`docker service create --name {{service_name}} {{image}}:{{tag}}`
```

Reply:
Done